scopes/scope/scope/scope.ui.runtime.tsx

```diff
@@ -258,8 +262,8 @@ export class ScopeUI {
   /**
    * register a sidebar link to the section above the drawers
    */
-  registerSidebarLink = (...links: SidebarItem[]) => {
-    this.sidebarItemSlot.register(links);
+  registerSidebarItems = (...items: SidebarItem[]) => {
```

Comment:
what is the difference between this registerSidebarLink?

Reply:
noticed that we called it registeringLinks even though the slot lets you register any item
calling it `links` seems incorrect, because it makes the consumer feel that the API will only accept Link components